File: Front/src/app/graphic-card/graphic.ts
import {ProductType} from "../shopping-cart/shopping-cart-item";

export interface Graphic {
  idGraf: number;
  model: string;
  uklad: string;
  cena: number;
  pamiec: string;
  rpamieci: string;
  zlacza: string;
  img: string;
  productType: ProductType;
}

File: Back/src/main/java/com/api/back/model/Graphic.java
package com.api.back.model;

import com.api.back.Service.ShoppingCart.Enum.ProductType;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Graphic {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(nullable = false, updatable = false)
    private Long idGraf;
    private String model;
    private String uklad;
    private float cena;
    private String pamiec;
    private String rPamieci;
    private String zlacza;
    private String img;
    @Enumerated(EnumType.STRING)
    private ProductType productType;
}
